code/game/gamemodes/miniantags/pulsedemon/pulsedemon.dm

```diff
@@ -767,9 +767,11 @@
 		return FALSE
 
 /mob/living/simple_animal/demon/pulse_demon/bullet_act(obj/item/projectile/proj)
-	if(istype(proj, /obj/item/projectile/ion))
```

Comment:
This doesn't remove their ion weakness, does it?

Reply:
Nope, ions have burn damage!